GameOfRunes/GameOfRunes/Utils/GameConfig.swift

```diff
@@ -135,6 +163,21 @@ enum GameConfig {
         static let horizontalOffSet: CGFloat = 100.0
         static let verticalOffSet: CGFloat = 100.0
         static let numEndPoints: Int = 3
+        
+        static func calculateHorizontallyDistributedPoints(
```

Comment:
can i check that this function is used to calculate the xposition of the sprite node based on the lane index?

Reply:
yup